fundamental_types.js

```diff
@@ -118,6 +118,8 @@ fundamental_types={
         "point":(buf)=> FundamentalHexType(buf,33).read(),
         "short_channel_id":(buf)=> ShortChannelIDType(buf).read(),
         "signature":(buf)=> FundamentalHexType(buf,64).read(),
+        "description":(buf)=> buf.toString('utf8'),
+        "signature":(buf)=> buf.toString('hex')
```

Comment:
These aren't necessary?  We already have signature above (you fix this in a later commit anyway) but there's no fundamental type called "description"?

I think you can just remove this now, since you fixed the bugs :)

Reply:
Yeah, it's a mistake wrote it while debugging some functions probably, will fix it in next commit.